src/main/java/uk/nhs/hee/trainee/details/model/PersonalDetails.java

```diff
@@ -22,78 +22,38 @@
 package uk.nhs.hee.trainee.details.model;
 
 import java.time.LocalDate;
-import java.util.ArrayList;
-import java.util.List;
-
 import lombok.AllArgsConstructor;
 import lombok.Builder;
 import lombok.Data;
 import lombok.NoArgsConstructor;
-import org.springframework.data.annotation.Id;
-import org.springframework.data.mongodb.core.index.Indexed;
-import org.springframework.data.mongodb.core.mapping.Document;
-import org.springframework.data.mongodb.core.mapping.Field;
 
-@Document(collection = "ContactDetails")
 @Data
+@Builder
 @NoArgsConstructor
 @AllArgsConstructor
-@Builder
-public class ContactDetails {
-  @Id
-  private String id;
+public class PersonalDetails {
```

Comment:
This class doesn't need @Builder and constructors annotation, does it?

Reply:
Removed @NoArgsConstructor, @AllArgsConstructor, @Builder
Thanks